pkg/util/testutil/flake/flake_test.go

```diff
@@ -0,0 +1,140 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.Datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package flake
+
+import (
+	"math/rand"
+	"sync"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type mockTesting struct {
+	*testing.T
+
+	mutex         sync.RWMutex
+	skipCallCount int
+	failCallCount int
+}
+
+func newMockTesting(t *testing.T) *mockTesting {
+	return &mockTesting{
+		T: t,
+	}
+}
+
+func (mt *mockTesting) Skip(_ ...any) {
+	func() {
+		mt.mutex.Lock()
+		defer mt.mutex.Unlock()
+		mt.skipCallCount++
+	}()
+	mt.SkipNow()
+}
+
+func (mt *mockTesting) Fail() {
+	mt.mutex.Lock()
+	defer mt.mutex.Unlock()
+	mt.failCallCount++
+}
+
+func (mt *mockTesting) SkipCount() int {
+	mt.mutex.RLock()
+	defer mt.mutex.RUnlock()
+	return mt.skipCallCount
+}
+
+func (mt *mockTesting) FailCount() int {
+	mt.mutex.RLock()
+	defer mt.mutex.RUnlock()
+	return mt.failCallCount
+}
+
+var trueValue = true
```

Comment:
```suggestion
const trueValue = true
```

Reply:
I use it to set a pointer, go does not allow getting the address of a const  https://stackoverflow.com/questions/35146286/find-address-of-constant-in-go